docs/rfc/rfc-1.adoc

```diff
@@ -503,37 +539,58 @@ this transaction.
 
 [[transaction-incentives]]
 === Transaction Incentives
-
-Governable Parameters:
-
-- `max_gas_refund_price`: The highest amount of gwei that the gas refund
-  contract will pay out per gas for a refund transaction.
-
-There are 7 main transactions that require incentives:
-
-==== Transactions That Must Be Submitted By Operators
-- Submit distributed key generation results for new wallet creation (weekly).
-- Signal that funds need to be moved between wallets if a <<heartbeat,heartbeat>> fails.
-- Signal that a staker is not performing their duty and should be marked as
-ineligible for rewards (as in the case that they were inactive during
-heartbeats).
-
-==== Transactions That Can Be Submitted By Anyone
-- Begin distributed key generation for new wallet creation (weekly).
-- Provide the proof for a <<sweeping,sweep>> transaction.
-- Provide the proof for a <<redemption,redemption>> transaction.
-- Provide the proof that funds have moved between wallets.
-
-To make sure that these transactions get posted, we need the DAO to fund an ETH
-pool that provides gas refunds for anyone submitting these transactions. We
-know the amount of gas spent, and we know the gas price used, and so can
-reimburse the operator.
-
-To prevent abuse, we need to let governance set caps (`max_gas_refund_price`)
-on how much eth can be reimbursed (so that a miner can't drain the pool).
-
-Further ideas are to run bots like those provided by https://www.gelato.network/[Gelato]
-in order to prevent problems like operators front-running each other.
+Transaction incentives are more deeply explored in link:rfc-6.adoc[RFC 6:
+Transaction Incentives]. Summarized:
+
+There are three different types of transactions: <<operator-only,Operator-Only>>,
+<<public-knowledge,Public-Knowledge>>, and <<punishment,Punishment>>.
+
+[[operator-only]]
+==== Operator-Only
+Operator-Only transactions are where only the operators have access to the
+information required to assemble the transaction with the right input
+parameters.
+
+In order to avoid all operators racing to submit the
+transaction at the same time, we have an off-chain gentleman's agreement to
+submit based on the operator's position in the group (can use the hash of the
+group's pubkey).
+
+If the designated operator does not submit their transaction before a timeout
+expires, the duty moves to the next operator and the group can sign a
+transaction to mark that operator as inactive. Such a transaction would be a
+<<punishment,Punishment>>.
```

Comment:
I am not sure if such a transaction should be _punishment_ or _operator only_, I'd rather say the latter.

First, I do not think we should reward for this transaction. ETH reimbursement is fine but the submitter should be net-0. We do not want to incentivize this call, while we do want incentivize _punishment_ transactions. Second, only operators from the given group can assemble this transaction because it requires a sufficient number of signatures to support the motion. Such a motion needs to be proposed off-chain and signers need to work off-chain to share their signatures supporting it.

Reply:
addressed in [6878b4a](https://github.com/keep-network/tbtc-v2/pull/127/commits/6878b4add75c025371166f7982465278ebfc626d)